libres/lib/res_util/block_fs.cpp

```diff
@@ -448,41 +444,41 @@ static void block_fs_open_data(block_fs_type *block_fs,
      2. The node is added to the block_fs instance as a free node, which can
         be recycled at a later stage.
 
-   If the instance is not data owner (i.e. read-only) the function
+   If the instance is read-only the function
    will return immediately.
 */
 static void block_fs_fix_nodes(block_fs_type *block_fs,
```

Comment:
name seems ominous. Are we using this?

Reply:
`block_fs_fix_nodes` is used in `block_fs_mount`.